draw_flip_book/lib/main.dart

```diff
@@ -79,31 +79,83 @@ class _FlipBookPageState extends State<FlipBookPage> {
     // fast, so that you can just rebuild anything that needs updating rather
     // than having to individually change instances of widgets.
     return new Scaffold(
-      body: buildGestureDetector(
+      body: _buildGestureDetector(
         context,
-        Stack(
-          alignment: Alignment.center,
-          children: <Widget>[
-            // TODO: Generalize/Scale
-            _buildPositionedFrame(context, key1, points1, true, Colors.green),
-            _buildPositionedFrame(
-                context, key2, points2, _isVisible2, Colors.lightBlue),
-            _buildPositionedFrame(
-                context, key3, points3, _isVisible3, Colors.amberAccent),
-          ],
+        Container(
+          child: Column(
+            mainAxisSize: MainAxisSize.min,
+            children: <Widget>[
+              Container(
+                height: 500,
+                child: _framesStack(context),
+              ),
+              Expanded(
+                child: Container(child: _buttonRow()),
+              )
+            ],
+          ),
         ),
       ),
-      floatingActionButton: FloatingActionButton(
-          onPressed: () {
-            setState(() {
-              _toggleFramesVisibility();
-            });
-          },
-          child: Icon(Icons.skip_next)),
     );
   }
 
-  GestureDetector buildGestureDetector(BuildContext context, Widget child) {
+  Row _buttonRow() {
+    return Row(
```

Comment:
Maybe break the containers in the children down to vars. So we know what each button does too.

Reply:
:+1: 6c0306d